dash/dash.py

```diff
@@ -29,8 +68,13 @@ def __init__(
             name='__main__',
             server=None,
             static_folder='static',
+            assets_folder=None,
+            assets_url_path='/assets',
+            include_assets_files=True,
             url_base_pathname='/',
             compress=True,
+            meta_tags=None,
+            index_string=_default_index,
             **kwargs):
```

Comment:
Let's also create an issue about adding a `css_urls` and `js_script_urls` arguments to this init argument as an alternative to our `app.css.append_css` and `app.scripts.append_script`:

> I actually like this declarative syntax. This seems like a nice pattern that we could use elsewhere, like as a way to replace app.scripts.append_script:

app = dash.Dash(
     js_script_urls=['https://cdn.google.com/google-analytics.js'],
     css_stylesheet_urls=['https://cdn.bootstrap.com/bootstrap.css']
)

> I never really liked the app.scripts.append_script syntax and I'd generally prefer for the dash.Dash() object to have as few methods and properties as possible. If everything could just be in the app.config and settable through the dash.Dash() constructor, I'd be very happy.

(from comment https://github.com/plotly/dash/pull/286/#issuecomment-404281839)